Update NearFutureSolar-Core for 1.1, primary moved to SpaceDock

lOmicronl opened the pull request.

lOmicronl commented:
Not sure why the build check failed. JSONLint says the syntax is valid...

plague006 commented:
You can't use `find` on a file without including `"find_matches_files": true` in the install stanza.

plague006 commented on NetKAN/NearFutureSolar-Core.netkan:
```diff
@@ -0,0 +1,17 @@
+    "name"                : "Near Future Solar Core",
+    "abstract"            : "This is the Near Future Solar plugin stand-alone, for adding functionality to other mods. It contains no parts and does nothing by itself.",
+    "author"              : "Nertea",
+    "license"             : "CC-BY-SA-NC-4.0",
```
`"CC-BY-NC-SA-4.0"`

lOmicronl commented:
Thanks, I think I fixed it by changing the "find" to a "file"... yep, the build suceeds now.

In regards to the license - how odd! I thought it was the wrong way around, because SpaceDock displays it differently on the mod pages, so I went and changed it everywhere. Guess CKAN didn't like that! ;)

plague006 commented:
Looks good, merging with my thanks.
